src/main/java/com/blackducksoftware/integration/hub/api/component/version/LicenseItem.java

```diff
@@ -23,37 +23,30 @@
  */
 package com.blackducksoftware.integration.hub.api.component.version;
 
-import org.apache.commons.lang3.builder.RecursiveToStringStyle;
-import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
-
 import com.blackducksoftware.integration.hub.api.item.HubItem;
-import com.blackducksoftware.integration.hub.api.item.HubResponse;
-
-public class License extends HubItem {
-	private final CodeSharingEnum codeSharing;
-	private final String name;
-	private final OwnershipEnum ownership;
-	
-	public License(CodeSharingEnum codeSharing, String name, OwnershipEnum ownership) {
-		this.codeSharing = codeSharing;
-		this.name = name;
-		this.ownership = ownership;
-	}
-
-	public CodeSharingEnum getCodeSharing() {
-		return codeSharing;
-	}
-
-	public String getName() {
-		return name;
-	}
-
-	public OwnershipEnum getOwnership() {
-		return ownership;
-	}
-	
-	@Override
-	public String toString() {
-		return ReflectionToStringBuilder.toString(this, RecursiveToStringStyle.JSON_STYLE);
-	}
+
+public class LicenseItem extends HubItem {
+    private final CodeSharingEnum codeSharing;
+
+    private final String name;
+
+    private final OwnershipEnum ownership;
+
+    public LicenseItem(CodeSharingEnum codeSharing, String name, OwnershipEnum ownership) {
```

Comment:
I suspect this constructor is not needed. If it is only used in tests, let's put the code in the test. If it is used elsewhere, that might smell of a more sinister issue.